common/persistence/src/main/resources/db/migration/V10__addBatchTagIdForUploadKeyTable.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE federation_upload_key
+    ADD batch_tag  VARCHAR(50);
```

Comment:
ensure this is later changed to varchar(64) - https://github.com/eu-federation-gateway-service/efgs-federation-gateway/blob/master/src/main/resources/db/changelog/v000-create-diagnosis-key-entity-table.yml#L24